src/app/page.tsx

```diff
@@ -1,27 +1,88 @@
-import { SignInButton, SignOutButton, currentUser} from '@clerk/nextjs'
+import { SignInButton, SignOutButton, currentUser } from '@clerk/nextjs';
 import Link from 'next/link';
+import { redirect } from 'next/navigation';
+import { ModeToggle } from '~/components/mode-toggle';
 import { Button } from '~/components/ui/button';
+import { buttonVariants } from "../components/ui/button";
+import dynamic from 'next/dynamic'; // Import dynamic from next/dynamic
+import { Hero } from '~/components/Hero';
+
+// Import MobileMenu dynamically to ensure it's rendered on the client-side
+const MobileMenu = dynamic(() => import('~/components/MobileMenu'), { ssr: false });
+
+interface RouteProps {
+  href: string;
+  label: string;
+}
+
+const routeList: RouteProps[] = [
+  {
+    href: "#features",
+    label: "Features",
+  },
+  {
+    href: "#testimonials",
+    label: "Testimonials",
+  },
+  {
+    href: "#pricing",
+    label: "Pricing",
+  },
+  {
+    href: "#faq",
+    label: "FAQ",
+  },
+];
 
 export default async function HomePage() {
   const user = await currentUser();
-  const isLoggedInn = user !== null;
+  const isLoggedIn = user !== null;
+
+  // // Redirect to /dashboard if user is logged in
```

Comment:
Whats going on with this commented out code?

Reply:
Hey great question, this should've been erased so my mistake here. I wrote this before I confirmed with you if still wanted users to have the ability to go back to the marketing page. This essentially would've ensured if isLoggedIn was true, it would use the redirect() function from next/navigation to take the user to the /dashboard instead. So long story short, this is not needed anymore.